rc/windowing/sway.kak

```diff
@@ -0,0 +1,67 @@
+provide-module sway %{
+
+# Ensure we're actually in Sway
+evaluate-commands %sh{
+    [ -z "${kak_opt_windowing_modules}" ] || 
+    [ "$DESKTOP_SESSION" = "sway" ] || 
+    [ "$XDG_SESSION_DESKTOP" = "sway" ] || 
+    echo 'fail DESKTOP_SESSION is not "sway"'
+}
+
+require-module 'wayland'
+
+define-command sway-focus-pid -hidden %{
+    evaluate-commands %sh{
+        # We need to walk our way up a tree of process IDs to find out which
+        # window a specific client is associated with. This can be further up
+        # than you might expect, as a window could be for a terminal emulator 
+        # that's running bash that's running kak.
+
+        focus_window_ancestor() {
+            # Get the container ID from the tree of windows
+            con_id=`swaymsg -t get_tree | jq ".. | (.nodes? // empty)[] | select(.pid == $1) | .id"`
+
+            if [ -z "$con_id" ]; then
+                # If we couldn't find a matching window, then we need to do
+                # the same check for our parent, so we grab our PPID
+                ppid=`ps -p $1 -o ppid=`
```

Comment:
Nit: I'd  use `$()` instead of backticks for consistency with other support scripts (same above)